lib/clusterable/ht_item.rb

```diff
@@ -26,16 +26,16 @@ class HtItem
     ACCESSOR_ATTRS.each { |attr| attr_accessor attr }
     READER_ATTRS.each { |attr| attr_reader attr }
 
-    def self.db
+    def self.table
```

Comment:
i like this change, db -> table. more accurately descriptive, less room for confusion

Reply:
I applied this elsewhere too.